workspace-manager-clienttests/src/main/java/scripts/testscripts/GetWorkspace.java

```diff
@@ -0,0 +1,48 @@
+package scripts.testscripts;
+
+import bio.terra.testrunner.runner.TestScript;
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import bio.terra.workspace.client.ApiClient;
+import bio.terra.workspace.api.WorkspaceApi;
+import bio.terra.workspace.client.ApiException;
+import bio.terra.workspace.model.WorkspaceDescription;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import scripts.utils.WorkspaceManagerServiceUtils;
+
+import java.util.UUID;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+public class GetWorkspace extends TestScript {
+    private static final Logger logger = LoggerFactory.getLogger(GetWorkspace.class);
+
+    /** Public constructor so that this class can be instantiated via reflection. */
+    public GetWorkspace() {
+        super();
+    }
+
+    @Override
+    public void userJourney(TestUserSpecification testUser) throws Exception {
+        ApiClient apiClient = WorkspaceManagerServiceUtils.getClientForTestUser(testUser, server);
+        WorkspaceApi workspaceApi = new WorkspaceApi(apiClient);
+
+        /**
+         * This GetWorkspace test expects a bogus workspace id resulting in
+         * workspace not found exception and http code 401
+         * **/
```

Comment:
As discussed in person, this test really needs to create a workspace in some `BeforeAll` hook, then attempt to get that workspace.

Reply:
Added setup and cleanup steps to GetWorkspace test.